config/htmlmin.php

```diff
@@ -46,4 +46,25 @@
 
     'force' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Ignore Blade Files
+    |--------------------------------------------------------------------------
+    |
+    | Here you can specify paths, which you don't want to minimize, for example
+    | e-mails or markdowns.
+    |
+    */
+
+    'ignore' => [
+        'resources/views/emails',
+        'resources/views/html',
+        'resources/views/notifications',
+        'resources/views/markdown',
+        'resources/views/vendor/emails',
+        'resources/views/vendor/html',
+        'resources/views/vendor/notifications',
+        'resources/views/vendor/markdown',
```

Comment:
@GrahamCampbell Can you please elaborate? `emails`, `html`, `notifications` and `markdown` all have `.blade.php` extensions (if I remember right), but all of these should be ignored naturally - and these can be dumped to the `resources/views/vendor/` folder from the framework's folder (`php artisan vendor:publish --tag=laravel-mail`), that's why `vendor` versions are ignored too. But maybe I just don't see what you mean.